WebAPI/nlog.Production.config

```diff
@@ -8,8 +8,11 @@
   <!-- https://github.com/nlog/NLog/wiki/Configuration-file -->
   <targets>
     <target xsi:type="File" name="file" fileName="/app/logs/${shortdate}.log"
-            layout="${longdate}|${level:uppercase=true}|${logger}|${message}" />
-  </targets>
+            layout="${longdate}|${level:uppercase=true}|${logger}|${message}"
+			archiveFileName="/app/logs/archive/${shortdate}.log"
+			maxArchiveFiles="14"
+			archiveEvery="Day"
```

Comment:
`archiveEvery="Day"` is only need when using static-filename. It will cause issues when already using dynamic FileName `fileName="/app/logs/${shortdate}.log"` that rolls automatically (Because of `${shortdate}`). See also https://github.com/NLog/NLog/wiki/File-target#dynamic-vs-static-archive-logic

Reply:
Thank you for your comment! We would need to find out hard way otherwise :).